### PR TITLE
feat(ONYX-1019): add artwork dimensions step

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Feat**

This PR solves [ONYX-1019]

### Description

Implement artwork dimensions step for the new submission flow
| Web | mWeb |
| --- | --- |
| image | image |

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ONYX-1019]: https://artsyproduct.atlassian.net/browse/ONYX-1019?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Sell/Routes/DimensionsRoute.tsx [hunk 1]
```diff
@@ -69,33 +75,54 @@ export const DimensionsRoute: React.FC<DimensionsRouteProps> = props => {
...
-              title="Height"
-              defaultValue={values.height}
-            ></Input>
-            <Input
-              onChange={handleChange}
-              name="depth"
-              title="Depth"
-              defaultValue={values.depth}
-            ></Input>
+          <Join separator={<Spacer y={4} />}>
```
Comment (olerichter00) [inline src/Apps/Sell/Routes/DimensionsRoute.tsx:78]: Just to make it easier to read :) 
```suggestion

          <Join separator={<Spacer y={4} />}>
```

File: src/Apps/Sell/Routes/DimensionsRoute.tsx [hunk 2]
```diff
@@ -25,10 +28,10 @@ const FRAGMENT = graphql`
 `
 
 const Schema = Yup.object().shape({
-  width: Yup.string().trim(),
-  height: Yup.string().trim(),
+  width: Yup.string().required().trim(),
+  height: Yup.string().required().trim(),
```
Comment (olerichter00) [inline src/Apps/Sell/Routes/DimensionsRoute.tsx:32]: Have we already made a decision on whether to make the dimensions required or not? Just found this [comment](https://www.figma.com/design/E6rVm1tiSI4PW3EzibKKZw?node-id=2036-30666#839058696) in Figma:

**Review by olerichter00 — APPROVED** [review]
Fantastic 🌟

File: src/Apps/Sell/Routes/DimensionsRoute.tsx (hunk 2 above)
Comment (dariakoko) [reply src/Apps/Sell/Routes/DimensionsRoute.tsx:32]: Ah, no, i don't think we have
Sorry, didn't see the comment, will remove the` required()`

**Review by olerichter00 — APPROVED** [review]

**Pull request merged**